molgenis-api-permissions/src/main/java/org/molgenis/api/permissions/model/request/DeletePermissionRequest.java

```diff
@@ -0,0 +1,19 @@
+package org.molgenis.api.permissions.model.request;
+
+import com.google.auto.value.AutoValue;
+import org.molgenis.util.AutoGson;
+
+@AutoValue
+@AutoGson(autoValueClass = AutoValue_DeletePermissionRequest.class)
+@SuppressWarnings(
+    "squid:S1610") // Abstract classes without fields should be converted to interfaces
+public abstract class DeletePermissionRequest {
+
+  public abstract String getUser();
+
+  public abstract String getRole();
+
+  public static DeletePermissionRequest create(String user, String role) {
+    return new AutoValue_DeletePermissionRequest(user, role);
```

Comment:
Validate that either user of role is non-null

Reply:
Fixed, commented line not changed so not marked as outdated.